db_scripts/createIdaNewDB.sh

```
@@ -8,32 +8,32 @@ if [[ $# -ne 3 ]]; then
 fi
 
 ## set variables
-oldDB=$1;     ## mosip_ida
-newDB=$2;     ## mosip_ida_1
+oldDB=$1;     ## :mosipdbname
+newDB=$2;     ## :mosipdbname_1
 dbPwd=$3
 
 ## create new DB directory
 rm -rf $newDB
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)